packages/babel-traverse/src/hub.js

```diff
@@ -1,6 +1,14 @@
+import Scope from "./scope";
+export interface HubInterface {
+  mark?: (type: string, message: string) => void;
+  addHelper?: (name: string) => Object;
+  getScope?: () => Scope;
+  getCode?: () => string;
+  buildError:(node: Object, msg: string, Error: Error) => Error;
+}
+
 export default class Hub {
-  constructor(file, options) {
-    this.file = file;
-    this.options = options;
+  buildError(node, msg, Error): Error {
```

Comment:
The coverage fails there because the method `buildError` isn't tested.

If you move the content of the object (as described above) there, we need to add tests for it.